test/language/computed-property-names/computed-property-names-setter-object.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    In an object, duplicate computed property getter names produce only a single property of
+    that name, whose value is the value of the last property of that name.
+---*/
+var calls = 0;
+var object = {
+  set ['a'](_) {
+    calls++;
+  }
+};
+object.a = 'A';
+assert.sameValue(calls, 1);
+
+calls = 0;
+object = {
+  set b(_) {
+    assert(false);
+  },
+  set ['b'](_) {
+    calls++;
+  }
+};
+object.b = 'B';
+assert.sameValue(calls, 1);
+
+calls = 0;
+object = {
+  set c(_) {
+    assert(false)
```

Comment:
Keep an eye on [this discussion](https://github.com/tc39/test262/pull/184#issuecomment-76986716); depending on its resolution, we may want to switch these to `$ERROR`